openjdk_regression/ProblemList_openjdk8-openj9.txt

```diff
@@ -20,7 +20,7 @@ java/lang/String/CaseInsensitiveComparator.java		https://github.com/eclipse/open
 java/lang/StringBuffer/Exceptions.java		https://github.com/eclipse/openj9/issues/1128	generic-all
 java/lang/StringBuffer/TestSynchronization.java		https://github.com/eclipse/openj9/issues/1128	generic-all
 java/lang/StringBuilder/Exceptions.java		https://github.com/eclipse/openj9/issues/1128	generic-all
-java/lang/System/Versions.java		https://github.com/eclipse/openj9/issues/1128	generic-all
+# java/lang/System/Versions.java		https://github.com/eclipse/openj9/issues/1128	generic-all fixed
```

Comment:
Rather than commenting out the line, please remove the line (same comment for both files).

Reply:
Thanks.  Done.